packages/esm-patient-chart-app/src/ui-components/notifications-button.component.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Button from 'carbon-components-react/es/components/Button';
+import { Notification20 } from '@carbon/icons-react';
+import styles from './action-menu.scss';
+
+interface NotificationsButtonInterface {
+  onClick: Function;
+}
+
+const NotificationsButton: React.FC<NotificationsButtonInterface> = ({ onClick }) => {
```

Comment:
just the example button to be used in the menu. I never received any requirements on what these buttons do so im just using this as an example